cluster/src/main/java/org/apache/iotdb/cluster/server/service/DataSyncService.java

```diff
@@ -352,8 +355,12 @@ public ByteBuffer getAllMeasurementSchema(RaftNode header, ByteBuffer planBinary
   public List<ByteBuffer> getAggrResult(GetAggrResultRequest request) throws TException {
     try {
       return dataGroupMember.getLocalQueryExecutor().getAggrResult(request);
-    } catch (StorageEngineException | QueryProcessException | IOException e) {
-      throw new TException(e);
+    } catch (StorageEngineException
+        | QueryProcessException
+        | IOException
+        | StatisticsClassException
+        | UnSupportedDataTypeException e) {
+      throw new TApplicationException(e.getMessage());
```

Comment:
The raw exception message will be shadowed by TException as it doesn't serialize the root cause to other nodes.